src/styles/header.scss

```diff
@@ -0,0 +1,16 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 60px;
```

Comment:
There is no need in hardcoding the `height` value, cause it could be done with the `padding` / `margin` values. It would be much easier later then to create responsive header